src/sprout/Controllers/Admin/ManagedAdminController.php

```diff
@@ -1546,11 +1552,11 @@ public function _editSave($item_id)
      * Optional custom HTML for the save box
      * Return NULL to use the default HTML
      *
-     * @param return string HTML
+     * @return string HTML
```

Comment:
While we're here, can we also add a type hint or the `$item_id` param?

Reply:
Good point, I added it to all duplicate methods. I don't have time to do the whole suite of admin methods in there though :D